src/providers/fail_over.c

```diff
@@ -1438,7 +1452,10 @@ resolve_srv_done(struct tevent_req *subreq)
     }
 
 done:
-    if (ret != EOK) {
+    if (ret == ERR_SRV_DUPLICATES) {
+        tevent_req_error(req, ret);
```

Comment:
It is still an error, since it is unexpected and we don't return requested data, only best effort. The caller decides how to handle it.